flexmeasures/api/v3_0/tests/test_sensor_data_fresh_db.py

```diff
@@ -50,3 +55,36 @@ def test_post_sensor_data(
     assert len(beliefs) == expected_num_values
     # check that values are scaled to the sensor unit correctly
     assert pytest.approx(beliefs[0].event_value - expected_value) == 0
+
+
+def test_get_sensor_data(
```

Comment:
I believe this could be in the non-fresh part of testing, as the test itself mutates no data. And you added the data points in both versions.

Reply:
I first implemented it in the non-fresh db test module, but it did interfere there. And I found it was trickier than just adapting the assert statements that check the number of beliefs registered on the sensor shared by these tests.